build.sbt

```diff
@@ -282,41 +312,46 @@ lazy val secp256k1jni = project
   */
 val testAndCompile = "compile->compile;test->test"
 
-lazy val cryptoTest = project
+lazy val cryptoTest = crossProject(JVMPlatform, JSPlatform)
+  .crossType(CrossType.Pure)
   .in(file("crypto-test"))
   .settings(CommonSettings.testSettings: _*)
+  .jsSettings(commonJsSettings)
   .settings(
     name := "bitcoin-s-crypto-test",
     libraryDependencies ++= Deps.cryptoTest.value
   )
   .dependsOn(
-    cryptoJVM % testAndCompile,
-    testkit
+    core,
```

Comment:
I see here a chicken-or-egg issue here. The test scope should be limited to `crypto` here